getting_started/step_by_step/7_room_two.rst

```diff
@@ -0,0 +1,211 @@
+Create a second room
+====================
+So a one room game is a good start, but your adventure game will be a lot more
+interesting when there's other rooms to travel to. Lets create a park scene
+and join it to the pub scene so we can walk between them.
+
+As we've already created the first scene, follow the same steps you used on the
+:doc:`create a room <3_create_room>`.  page to create this scene.
+
+* Create a new folder in your rooms folder and call it "park"
+* Create a new Godot scene named "room02" and set ``ESCRoom`` as its root node.
+  * Note that you could call it something other than room02, we're just being
+  consistent with the first room.
+* Give the room a global id (``park``)
+* We still want to play as Graham in this scene, so select it as the
+  ``player scene`` parameter.
+* Add an ESCBackground node for the room graphics as a child of the root node.
+* From ``backgrounds.zip``, copy the ``park_bg.png`` file to the **rooms**
+  directory and set it as the ``texture`` parameter in the ``ESCBackground``
+  node.
+* Create a walkable area for the character. Add an ``ESCTerrain``
+  node to the ``ESCRoom`` node. Add a
+  ``NavigationPolygonInstance`` node as a child of the new ``ESCTerrain`` node.
+* Create a NavigationPolygonInstance resource for the Navpoly parameter.
+
+.. image:: img/create_room_terrain_navpoly2.png
+   :alt: Adding a NavPoly
+
+* Use the editor window to draw a polygon. This polygon is the area in which
+  the character can move around freely.
+
+.. image:: img/create_room_terrain_createdpoly2.png
+   :alt: View of the completed polygon on the background
+
+Creating a light map
+--------------------
+
+For this room we're going to use a light map. As mentioned previously, light
+maps are used to shade different areas of the room differently. In this room,
+we have a lamp, so we want the player to be brightly lit when under it, and
+less brightly lit when away from the light source.
+
+The example light map provided is a very simple one with just 3 colours to make
+it easier to demonstrate how the light map changes how the game character
+looks, but you could replace it with one that uses a gradient to fade nicely
+from white to mid-grey. Coloured ones can also be used to simulate things like
+concert or traffic lights.
+
+Copy the park_lightmap.png asset (link on the main step-by-step page) to the
+room02 folder.
+
+In the Godot GUI, select the ``ESCTerrain`` node, and set its Lightmap to the
+park_lightmap.png texture.
+
+Create a doorway back to the pub room
+-------------------------------------
+To link room 2 (the park) to room 1 (the pub), you will need an ESCexit. When
```

Comment:
I'd link ESCExit to the API doc page here (see other pages for the syntax for this)